rules/utils.js

```diff
@@ -33,7 +33,41 @@ function isjQuery(node) {
   return id && id.name.startsWith('$')
 }
 
+function withProperty(property, callback) {
+  if (typeof callback !== 'function') {
+    throw new TypeError('Must provide callback function')
+  }
+
+  const properties = new Set([].concat(property))
+
+  return function(node) {
+    if (node.callee.type !== 'MemberExpression') return
+    if (node.callee.object.name !== '$') return
```

Comment:
In regards to #18, if we pass the ESLint `context` into the `withProperty()` or `withMethod()` helpers, we can refactor this line to support jQuery global aliases:

```js
// pseudocode
const aliases = getjQueryAliases(context)
if (!aliases.has(node.callee.object.name)) return
```